Formula/csound.rb

```diff
@@ -68,31 +84,45 @@ def install
 
     libexec.install "#{buildpath}/interfaces/ctcsound.py"
 
-    version = Language::Python.major_minor_version "python3"
-    (lib/"python#{version}/site-packages/homebrew-csound.pth").write <<~EOS
+    python_version = Language::Python.major_minor_version "python3"
+    (lib/"python#{python_version}/site-packages/homebrew-csound.pth").write <<~EOS
       import site; site.addsitedir('#{libexec}')
     EOS
+
+    (pkgshare/"hrtf-data").install Dir["samples/*.dat"]
```

Comment:
If this is suitable to install, it should be installed by `make install`.

Reply:
Understood, I’ll open an issue upstream.